qt-models/divesummarymodel.cpp

```diff
@@ -121,6 +121,10 @@ static void calculateDive(struct dive *dive, Stats &stats)
 		return;
 	}
 
+	if (dive->invalid) {
+		return;
+	}
+
```

Comment:
could you provide some details why this is needed in the PR description?
i had to remind myself what it means for a dive to be invalid.